README.md

```diff
@@ -230,6 +230,23 @@ See: https://stackoverflow.com/questions/50168647/multiprocessing-causes-python-
 
 ## Contribution Guide
 
+### Creating and Publishing the Collection
+
+To build the collection you first need to bump the version number listed in ansible/redpanda/cluster/galaxy.yml
```

Comment:
have you had a chance to write up the version guidance? e.g., how/when we make a major vs minor change?

Reply:
My assumption was standard practice for tech companies: ~~we just pick a number out of a hat~~ following the doctrine listed here: https://semver.org/